webextensions/api/declarativeNetRequest.json

```diff
@@ -969,6 +969,26 @@
               }
             }
           },
+          "isUrlFilterCaseSensitive": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "84",
+                  "notes": "From 118 default value is false."
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "113"
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "16.4"
```

Comment:
Has Safari supported the agreed upon default starting from the first release...?

@xeenon What is the version where this property was introduced, and if different, what is the version where the default matches the "standard"?